.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,19 @@
+name: golangci-lint
```

Comment:
maybe we can add a `.golangci.yml`?
```
linters-settings:
  funlen:
    lines: 100
    statements: 75
  gocritic:
    disabled-checks:
      - exitAfterDefer

linters:
  disable:
    - errcheck
  enable:
    - bodyclose
    - deadcode
    - depguard
    - dogsled
    - dupl
    - exhaustive
    - funlen
    - gochecknoinits
    - goconst
    - gocritic
    - gocyclo
    - gofmt
    - goimports
    - golint
    - goprintffuncname
    - gosimple
    - govet
    - ineffassign
    - misspell
    - nakedret
    - noctx
    - nolintlint
    - rowserrcheck
    - scopelint
    - staticcheck
    - structcheck
    - stylecheck
    - typecheck
    - unconvert
    - unparam
    - unused
    - varcheck
    - whitespace
```

Reply:
Not sure... It works without a config well enough and I don't want to learn & maintain golangci-lint config :) If you already have a config that you use in other projects - please send a PR.